frontend/discourse/app/models/composer.js

```diff
@@ -961,6 +961,7 @@ export default class Composer extends RestModel {
       archetypeId: opts.archetypeId || this.site.default_archetype,
       metaData: opts.metaData ? EmberObject.create(opts.metaData) : null,
       reply: opts.reply || this.reply || "",
+      originalText: opts.reply || this.reply || "",
```

Comment:
This was the fix, we were always considering the draft "dirty" when opening it even when nothing changed because we were not comparing against the originalText:

https://github.com/discourse/discourse/blob/573d514e0dec62e30fb28c5a3f2dbd201072d957/frontend/discourse/app/models/composer.js#L250-L253